ccm.css

```
@@ -138,7 +138,8 @@
         position: relative;
         transition: opacity 200ms ease;
 
-        [data-action=playCard] {
+        [data-action=playCard],
```

If we change this, let's at least be consistent.

```suggestion
        [data-action="playCard"],
```